.github/workflows/breaking-change-description-title-update.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 # cancel redundant builds
 concurrency:
-  group: "update-breaking-change-${{ github.ref }}"
+  group: "update-breaking-change-${{ github.head_ref }}"
```

Comment:
Not sure that's what you want here, from https://docs.github.com/en/actions/learn-github-actions/contexts ref should be the branch name

Reply:
This was what we had earlier; it seems like for `pull_request_target` ref defaults to `main` while for `pull_request` it is unique to the PR. So this job doesn't need head_ref but the pull_request_target job (title check does). Reverting this.